src/foam/nanos/notification/NotificationCitationView.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```

Comment:
2020

Reply:
How is this different than the NotificationNotifcationView?

Reply:
NotificationNotificationView simply extends the NotificationCitationView, I don't see the point of it and I don't think its used anywhere.